inference-engine/src/auto_plugin/auto_plugin.cpp

```diff
@@ -38,6 +38,38 @@ AutoInferencePlugin::AutoInferencePlugin() {
     _pluginName = "AUTO";
 }
 
+IE::IExecutableNetworkInternal::Ptr AutoInferencePlugin::LoadNetwork(const std::string& fileName,
```

Comment:
if the only diff with the LoadNEtworkExeImpl is filename passed to the core (as opposite to the cnnNetwork), maybe we need a better solution than code duplication

Reply:
Agree. It's already fixed in AUTO plugin related PR. Suggest to merge as is and that PR will fix this